azure/iac/sql_servers.rego

```diff
@@ -16,10 +23,19 @@ azure_issue["sql_public_access"] {
 
 sql_public_access {
     lower(input.resources[_].type) == "microsoft.sql/servers"
+    not azure_attribute_absence["sql_public_access"]
     not azure_issue["sql_public_access"]
 }
 
 sql_public_access = false {
+    azure_attribute_absence["sql_public_access"]
```

Comment:
this should be `true `with name `sql_public_access_disabled`

Reply:
I didn't understand what is the issue about it
can you please explain it a little more?!
@rezoan

Reply:
fixed